static/defaultPage.js

```diff
@@ -1,48 +1,72 @@
+var newOrLoad;
+
 var startGame = function() {
-    slug = document.getElementById('gameNameInput').value
-    gamePassword = document.getElementById('gamePasswordInput').value
-    localStorage.setItem('pwd_' + slug, gamePassword)
-    localStorage.setItem('slug', slug)
+    var slug = document.getElementById('gameNameInput').value
+    var gamePassword = document.getElementById('gamePasswordInput').value
     if (slug == '') {
         alert('Enter a game name');
         return;
     }
-    callback = function(pass) {
+    if (gamePassword == '') {
+        alert('Enter a password');
+        return;
+    }
+    localStorage.setItem('pwd_' + slug, gamePassword)
+    localStorage.setItem('slug', slug)
+    var callback = function(pass) {
         window.location.href = '/game/#' + slug;
     };
     var data = {
         password: gamePassword,
+        newOrLoad: newOrLoad
     };
     sendToServer('game-state/' + slug, data, callback);
 };
 
-var toggleIDDisplay = function(idName) {
-    if (document.getElementById(idName).style.display == "none") {
-        document.getElementById(idName).style.display = "block";
-    }
-    if (document.getElementById(idName).style.display == "block") {
-        document.getElementById(idName).style.display = "none";
-    }
+var newGame = function() {
```

Comment:
newGame and loadGame do a lot of the same thing.  Pulling this out to its own function would make this code a lot less verbose.

If you take in a value saying whether you're hiding or showing the main menu, you can use it in your mainMenu function too:

```
var updateMenu(showMainMenu) {
  var mainDisplay = "none";
  var newLoadDisplay = "block";
  if (showMainMenu) {
    mainDisplay = "block";
    newLoadDisplay = "none";
  }
  ...
}
```